inc/spbc-auth.php

```diff
@@ -486,6 +489,50 @@ function spbc_2fa__key_remove_old()
     $spbc->save('data');
 }
 
+/**
+ * Handle for check if rate limit is passed
+ *
+ * @return bool
+ */
+function spbc_2fa_is_rate_limit_pass()
+{
+    $rateLimit = get_option('spbc_rate_limit_2fa', [
+        'limit' => 10,
+        'expires_in' => time() + 60,
```

Comment:
Every time when you call time() function result could be different depends on system performance.

Better way is to init a variable like $current_time and do all comparison with this static variable during the function spbc_2fa_is_rate_limit_pass() call.

Reply:
Fixed https://github.com/CleanTalk/security-malware-firewall/pull/370/commits/0a977c9a46ec058ff7a2e6e9012d8673168e218b